htdocs/projet/element.php

```diff
@@ -565,6 +563,8 @@
 				$tmp=explode('_',$elementarray[$i]);
 				$idofelement=$tmp[0];
 				$idofelementuser=$tmp[1];
+        
+        $element = new $classname($db);
```

Comment:
If the loop has 1000 record, you will make 1000 new time a new instance when we can use only 1 for all the page (saving memory and cpu).
Is this really required ?